models/standalone-price/src/standalone-price-draft/generator.ts

```diff
@@ -21,7 +20,7 @@ const generator = Generator<TStandalonePriceDraft>({
     channel: fake(() => KeyReferenceDraft.presets.channel()),
     validFrom: fake(getValidFrom),
     validUntil: fake(getValidUntil),
-    tiers: [fake(() => PriceTierDraft.random())],
+    tiers: null,
```

Comment:
Why don't we keep the current behavior? We might be breaking existing tests due to this change.

Reply:
currencies in the tiers has to match main price to fit the spec. Either we mutate it on the transform phase, or like so is going to be an invalid price 50% of the time. We could bump a major version, but doubt anyone else is using this entity. B2B data set is not affected by this.